index.php

```diff
@@ -14,15 +14,20 @@
 $key = strval( getenv( 'KEY' ) );
 
 $auth = new APNSCredentials( $key_id, $team_id, $key );
-$configuration = APNSConfiguration::production( $auth );
+$configuration = APNSConfiguration::sandbox( $auth );
 $configuration->setUserAgent( strval( getenv( 'USER_AGENT' ) ) );
 $client = new APNSClient( $configuration );
-$client->setDebug( true );
+$client->setDebug( false );
```

Comment:
This should probably be `true` in this particular example, otherwise the line doesn't make much sense

Reply:
Yeah, good point. Besides, `index.php` is for local testing, so it makes sense to have the client running in debug mode.